releasenotes/notes/kafka-lag-support-data-streams-c6b7aed85bf2cf14.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    data_streams: Start tracking Kafka lag in seconds.
```

We should use third person present tense here ([docs](https://ddtrace.readthedocs.io/en/stable/releasenotes.html#style-guidelines)). 

Also can we be more explicit about the customer impact of this feature. Does this power a certain dashboard/view? Whats the minimum time resolution? If the kafka lag less than 1 second, is this value rounded to zero?
 
```suggestion
    data_streams: Starts tracking Kafka lag in seconds.
```